pkgs/development/python-modules/collections-extended/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+}:
+buildPythonPackage rec {
+  pname = "collections-extended";
+  version = "2.0.0";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256:16vyx8c5glqdyzjjpfwqp1cx8jlcshh0pi85n9mg8475ah07h7b5";
+  };
+
+  # As of 2.0.0 the tests aren't in the distribution archive.
+  doCheck = false;
```

Comment:
I've been asked in previous PR to switch to the GitHub archive if the tests are not available in the PyPi release, and use `pythonImportsCheck`.